include/zephyr/net/wifi.h

```diff
@@ -15,13 +15,28 @@
 enum wifi_security_type {
 	WIFI_SECURITY_TYPE_NONE = 0,
```

Comment:
Intentionally left out WEP as no real user uses it, and its only needed for WFA certification which will use a different shell not this API. And WPA3 is already there `WIFI_SECURITY_TYPE_SAE` is WPA3.

Reply:
Well the WEP is mentioned in Matter spec. Maybe it will be possible to filter it out based on what we've got. Sorry I was meaning WPA vs. WPA2, not WPA3.

Reply:
@markaj-nordic @krish2718 Let's discuss this offline. I think there are some pieces of a solution for what Matter team needs already in Zephyr and NCS.